### PR TITLE
Skill migrations

### PR DESCRIPTION
**Pull request by realalexhomer**
This branch adds migrations so that we can get the staging and production databases up to speed.

### CODE REVIEW

File: migrations/5.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Author: alex
```
Comment (jackofseattle) [inline migrations/5.js]: Why are there 2 migration files?  You can do both sets of mutations in a single migration file, that makes it easier to effectively rollback changes.

Comment (realalexhomer) [inline migrations/5.js]: You have to return a promise in the migrations or they won't work. Should I try to wrap them into one?

Comment (jackofseattle) [inline migrations/5.js]: Yeah, either with a Promise.all() or just chain them.

Comment (jshor) [inline migrations/5.js]: i agree that they should probably be wrapped into one

Comment (realalexhomer) [inline migrations/5.js]: Thanks I didn't know about Promise.all

**Pull request merged**